AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/GetAllocationHearingsTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using BookingsApi.Client;
+using AdminWebsite.Contracts.Responses;
+using Autofac.Extras.Moq;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Contract.Responses;
+
+namespace AdminWebsite.UnitTests.Controllers.HearingsController;
+
+public class GetAllocationHearingsTests
+{
+    private AutoMock _mocker;
+    private AdminWebsite.Controllers.HearingsController _controller;
+
+    [SetUp]
+    public void Setup()
+    {
+        _mocker = AutoMock.GetLoose();
+        _controller = _mocker.Create<AdminWebsite.Controllers.HearingsController>();
+    }
+
+    [Test]
+    public async Task Should_get_allocation_hearings()
+    {
+        // Arrange
+        _mocker.Mock<IBookingsApiClient>().Setup(client => client.SearchForAllocationHearingsAsync(
```

Comment:
may I ask what's an allocation hearing?

Reply:
A hearing for the allocation table. It can be either allocated or unallocated